tools/dev-doctor/rootcmd.go

```diff
@@ -187,7 +187,7 @@ func rootCmdRun(cmd *cobra.Command, args []string) {
 				ifNotFound:    checkError,
 				versionArgs:   []string{"--version"},
 				versionRegexp: regexp.MustCompile(`hub\s+version\s+(\d+.\d+\.\d+)`),
-				minVersion:    &semver.Version{Major: 2, Minor: 0, Patch: 0},
+				minVersion:    &semver.Version{Major: 2, Minor: 14, Patch: 0},
```

Comment:
While at it, shall we also suggest steps to install this particular version?

Reply:
Good idea - added hint.